question.html

```diff
@@ -47,37 +47,15 @@
 		</div>
 		<div id="navbar" class="navbar-collapse collapse">
 			<ul class="nav navbar-nav">
-				<p class="navbar-text"><b>Ask, Answer, Learn.</b></p>
+				<p class="navbar-text">Question Room: <b>{{roomId}}</b></p>
 			</ul>
 			<ul class="nav navbar-nav navbar-right">
-				<li><a href="index.html" style="font-family: 'Bubblegum Sans', cursive">Home!</a></li>
-				<li>
-					<button type="button" class="btn btn-default navbar-btn" ng-hide="$authData" ng-click="FBLogin()">
-						Sign in with Facebook!
-					</button>
-				</li>
+				<li><a href="index.html">Change Room</a></li>
 			</ul>
 		</div><!--/.nav-collapse -->
 	</div>
 </nav>
 
-<div class="container">
```

Comment:
Agree, the removal of jumbotron clears the page view. Also removes the share room button which is not functioning right now.